docs/available-filters/oauthbearer/oauthbearer.adoc

```diff
@@ -0,0 +1,30 @@
+= OAUTHBEARER pre-validation
+
+== What is it?
+
+OauthBearer filter enables a validation on the JWT token received from client before forwarding it to upstream.
+
+If the token is not validated, then the request is short-circuited.
+It permits to reduce resource consumption on the upstream when a client send too many invalid SASL requests.
+
+There are two steps to using the filter.
+
+1. <<Configuring virtual clusters>>
+2. Configuring the filter within Kroxylicious.
+
+=== Configuring the filter within Kroxylicious.
+
+[source, yaml]
+----
+filters:
+  - type: OauthBearer
+    config:
+      jwksEndpointUrl: https://oauth/JWKS   #<1>
+      jwksEndpointRefreshMs: 3600000        #<2>
+      jwksEndpointRetryBackoffMs: 100       #<3>
+      jwksEndpointRetryBackoffMaxMs: 10000  #<4>
+      scopeClaimName: scope                 #<5>
+      subClaimName: sub                     #<6>
+----
+
+<1> The OAuth/OIDC provider URL from which the provider's JWKS (JSON Web Key Set) can be retrieved
```

Comment:
I assume you forget to define the `<2>` ... `<6>`?

Reply:
Yes, still in progress :)